HPXMLtoOpenStudio/resources/schedules.rb

```diff
@@ -1,11 +1,49 @@
 # frozen_string_literal: true
 
+# Annual constant schedule
+class ScheduleRulesetConstant
```

Comment:
Create this new schedule class to be used in place of ScheduleConstant (where we can't set any outage rules).